### ISSUE TITLE
Error while decoding JWT token

### ISSUE THREAD

**Issue by GaetanFauconnier85** [dsferruzza/actix-web-middleware-keycloak-auth#2]
Hello, 

The token is not correctly decoded when there is more than 2 resource_access in addition to the default one in the keycloak token.

Steps to reproduce

Api part:

    Implement like in the library example.

Keycloak part:

    Create a realm on keycloak. 
    Create a admin role and an user associate to it (admin-user). Create a 2nd role (member) and associate it to a new user       (member-user).
    Create a keycloak client.
    Create 2 role in the client: client-admin and client-member
    Go to the admin-user and go to role-mapping. In the client-role select, pick the client you juste created and assign the client-admin role.
    Go to the member-user and go to role-mapping. In the client-role select, pick the client you juste created and assign the client-member role.

Postman part:

   Get the token by calling the keycloak api (by user) and make a API call with this token.

Current behaviour (bug)

   `Error while decoding JWT token (JSON error: invalid type: sequence, expected a string at line 1 column 146)`

Expected behaviour (correct)

    The token should be correctly decoded.

Relevant logs and/or media (optional)

`[2020-12-11T15:19:39Z DEBUG actix_web_middleware_keycloak_auth] Bearer token was extracted from request headers
[2020-12-11T15:19:39Z DEBUG actix_web_middleware_keycloak_auth] JWT header was decoded
[2020-12-11T15:19:39Z DEBUG actix_web_middleware_keycloak_auth] JWT is using RS256 algorithm
[2020-12-11T15:19:39Z DEBUG actix_web_middleware_keycloak_auth] Error while decoding JWT token (JSON error: invalid type: sequence, expected a string at line 1 column 146)
[2020-12-11T15:19:39Z INFO  actix_web::middleware::logger] 127.0.0.1:57949 "GET /private/users/1 HTTP/1.1" 401 107 "-" "PostmanRuntime/7.26.8" 0.000597
`

Working with one ressource_acces

<img width="328" alt="image" src="https://user-images.githubusercontent.com/36500464/102085469-f6750980-3e16-11eb-94d0-78f42e58160c.png">

Not working if there is more than 2 ressource_acces

<img width="333" alt="image" src="https://user-images.githubusercontent.com/36500464/102085626-32a86a00-3e17-11eb-9365-b638a6ede676.png">

Reported by

Gaëtan Fauconnier

**Comment by dsferruzza** [comment]
Hi!
Thanks for the report!

Unfortunately I didn't manage to reproduce this issue yet.
I tried the following steps:

1. run the example app with `cargo run --example simple`
2. generate a JWT using the private RS256 key I provided in the example source code and multiple objects inside `resource_access` ([see the token](https://jwt.io/#debugger-io?token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIzYjA4MDU3ZC03ODJmLTRiMTktYmQxNi02OWZmMjE5ZTc4M2YiLCJleHAiOjE3MDc5NjIwNzIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJ0ZXN0Il19LCJyZXNvdXJjZV9hY2Nlc3MiOnsiY2xpZW50MSI6eyJyb2xlcyI6WyJ0ZXN0MSJdfSwiY2xpZW50MiI6eyJyb2xlcyI6WyJ0ZXN0MiJdfX0sImlzcyI6ImFjdGl4LXdlYi1taWRkbGV3YXJlLWtleWNsb2FrLWF1dGgiLCJhdWQiOiJhY2NvdW50IiwiaWF0IjoxNjA3OTYyMDEyLCJqdGkiOiIxMTExMTExMS0xMTExLTExMTEtMTExMS0xMTExMTExMTExMTEiLCJhenAiOiIifQ.OJX31NLeJWz1Y8o81zQ-QTQpZrxRqwH2ZcqTfbH3LYLoej_7J808VDa-Quh3MFC3fs7VhOzqnnG31q-1SmQ59ni01XAtvUSVmhul6So4C86tyh2q_J0-g-sdQqdkxqbxqamMZ7h13gdz6oM_tvv_dl6T8jsyeF4CSwmNyMbA1OqNCt7K6ogrROtdqnXNOZhIVdU6L4QvSOuWbVFUuOv5A-WC7ahYITH9Bvguwh9ga6jtGklpO1kYFHuqQLC1HiywxaBBBGaP_lCFjc1lG-7nvD90jJq3hypErzTJnZHP3EeTSGCN0c34ND0Pf7LTqlnJYTItAbP5zKKAsEJ_W63mPg&publicKey=-----BEGIN%20PUBLIC%20KEY-----%0AMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnzyis1ZjfNB0bBgKFMSv%0AvkTtwlvBsaJq7S5wA%2BkzeVOVpVWwkWdVha4s38XM%2Fpa%2Fyr47av7%2Bz3VTmvDRyAHc%0AaT92whREFpLv9cj5lTeJSibyr%2FMrm%2FYtjCZVWgaOYIhwrXwKLqPr%2F11inWsAkfIy%0AtvHWTxZYEcXLgAXFuUuaS3uF9gEiNQwzGTU1v0FqkqTBr4B8nW3HCN47XUu0t8Y0%0Ae%2Blf4s4OxQawWD79J9%2F5d3Ry0vbV3Am1FtGJiJvOwRsIfVChDpYStTcHTCMqtvWb%0AV6L11BWkpzGXSW4Hv43qa%2BGSYOD2QU68Mb59oSk2OB%2BBtOLpJofmbGEGgvmwyCI9%0AMwIDAQAB%0A-----END%20PUBLIC%20KEY-----))
3. run `curl -v http://localhost:8080/private -H 'Authorization: Bearer [TOKEN]'`

With this scenario, I get the following output which seems fine:

```
ReqData(Claims { sub: 3b08057d-782f-4b19-bd16-69ff219e783f, exp: 2024-02-15T01:54:32Z, realm_access: Some(Access { roles: ["test"] }), resource_access: Some({"client1": Access { roles: ["test1"] }, "client2": Access { roles: ["test2"] }}), iss: "actix-web-middleware-keycloak-auth", aud: Some("account"), iat: 2020-12-14T16:06:52Z, jti: 11111111-1111-1111-1111-111111111111, azp: "" })
```

Any idea of what I am missing?
Could you provide me a valid JWT that does not work?
_Use the Share button in https://jwt.io/ like I did. You can also set the pub/priv keys as the ones I gave in my example, get the JWT payload from your Keycloak, and increase the `exp` value just to be sure._

**Comment by GaetanFauconnier85** [comment]
https://jwt.io/#debugger-io?token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIzYjA4MDU3ZC03ODJmLTRiMTktYmQxNi02OWZmMjE5ZTc4M2YiLCJleHAiOjE3MDc5NjIwNzIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJ0ZXN0Il19LCJyZXNvdXJjZV9hY2Nlc3MiOnsiY2xpZW50MSI6eyJyb2xlcyI6WyJ0ZXN0MSJdfSwiY2xpZW50MiI6eyJyb2xlcyI6WyJ0ZXN0MiJdfX0sImlzcyI6ImFjdGl4LXdlYi1taWRkbGV3YXJlLWtleWNsb2FrLWF1dGgiLCJhdWQiOiJhY2NvdW50IiwiaWF0IjoxNjA3OTYyMDEyLCJqdGkiOiIxMTExMTExMS0xMTExLTExMTEtMTExMS0xMTExMTExMTExMTEiLCJhenAiOiIifQ.OJX31NLeJWz1Y8o81zQ-QTQpZrxRqwH2ZcqTfbH3LYLoej_7J808VDa-Quh3MFC3fs7VhOzqnnG31q-1SmQ59ni01XAtvUSVmhul6So4C86tyh2q_J0-g-sdQqdkxqbxqamMZ7h13gdz6oM_tvv_dl6T8jsyeF4CSwmNyMbA1OqNCt7K6ogrROtdqnXNOZhIVdU6L4QvSOuWbVFUuOv5A-WC7ahYITH9Bvguwh9ga6jtGklpO1kYFHuqQLC1HiywxaBBBGaP_lCFjc1lG-7nvD90jJq3hypErzTJnZHP3EeTSGCN0c34ND0Pf7LTqlnJYTItAbP5zKKAsEJ_W63mPg&publicKey=-----BEGIN%20PUBLIC%20KEY-----%0AMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnzyis1ZjfNB0bBgKFMSv%0AvkTtwlvBsaJq7S5wA%2BkzeVOVpVWwkWdVha4s38XM%2Fpa%2Fyr47av7%2Bz3VTmvDRyAHc%0AaT92whREFpLv9cj5lTeJSibyr%2FMrm%2FYtjCZVWgaOYIhwrXwKLqPr%2F11inWsAkfIy%0AtvHWTxZYEcXLgAXFuUuaS3uF9gEiNQwzGTU1v0FqkqTBr4B8nW3HCN47XUu0t8Y0%0Ae%2Blf4s4OxQawWD79J9%2F5d3Ry0vbV3Am1FtGJiJvOwRsIfVChDpYStTcHTCMqtvWb%0AV6L11BWkpzGXSW4Hv43qa%2BGSYOD2QU68Mb59oSk2OB%2BBtOLpJofmbGEGgvmwyCI9%0AMwIDAQAB%0A-----END%20PUBLIC%20KEY-----

**Comment by dsferruzza** [comment]
That's weird, because it seems to work:

```
$ curl -v http://localhost:8080/private -H 'Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIzYjA4MDU3ZC03ODJmLTRiMTktYmQxNi02OWZmMjE5ZTc4M2YiLCJleHAiOjE3MDc5NjIwNzIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJ0ZXN0Il19LCJyZXNvdXJjZV9hY2Nlc3MiOnsiY2xpZW50MSI6eyJyb2xlcyI6WyJ0ZXN0MSJdfSwiY2xpZW50MiI6eyJyb2xlcyI6WyJ0ZXN0MiJdfX0sImlzcyI6ImFjdGl4LXdlYi1taWRkbGV3YXJlLWtleWNsb2FrLWF1dGgiLCJhdWQiOiJhY2NvdW50IiwiaWF0IjoxNjA3OTYyMDEyLCJqdGkiOiIxMTExMTExMS0xMTExLTExMTEtMTExMS0xMTExMTExMTExMTEiLCJhenAiOiIifQ.OJX31NLeJWz1Y8o81zQ-QTQpZrxRqwH2ZcqTfbH3LYLoej_7J808VDa-Quh3MFC3fs7VhOzqnnG31q-1SmQ59ni01XAtvUSVmhul6So4C86tyh2q_J0-g-sdQqdkxqbxqamMZ7h13gdz6oM_tvv_dl6T8jsyeF4CSwmNyMbA1OqNCt7K6ogrROtdqnXNOZhIVdU6L4QvSOuWbVFUuOv5A-WC7ahYITH9Bvguwh9ga6jtGklpO1kYFHuqQLC1HiywxaBBBGaP_lCFjc1lG-7nvD90jJq3hypErzTJnZHP3EeTSGCN0c34ND0Pf7LTqlnJYTItAbP5zKKAsEJ_W63mPg'
*   Trying ::1:8080...
*   Trying 127.0.0.1:8080...
* Connected to localhost (127.0.0.1) port 8080 (#0)
> GET /private HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.73.0
> Accept: */*
> Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIzYjA4MDU3ZC03ODJmLTRiMTktYmQxNi02OWZmMjE5ZTc4M2YiLCJleHAiOjE3MDc5NjIwNzIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJ0ZXN0Il19LCJyZXNvdXJjZV9hY2Nlc3MiOnsiY2xpZW50MSI6eyJyb2xlcyI6WyJ0ZXN0MSJdfSwiY2xpZW50MiI6eyJyb2xlcyI6WyJ0ZXN0MiJdfX0sImlzcyI6ImFjdGl4LXdlYi1taWRkbGV3YXJlLWtleWNsb2FrLWF1dGgiLCJhdWQiOiJhY2NvdW50IiwiaWF0IjoxNjA3OTYyMDEyLCJqdGkiOiIxMTExMTExMS0xMTExLTExMTEtMTExMS0xMTExMTExMTExMTEiLCJhenAiOiIifQ.OJX31NLeJWz1Y8o81zQ-QTQpZrxRqwH2ZcqTfbH3LYLoej_7J808VDa-Quh3MFC3fs7VhOzqnnG31q-1SmQ59ni01XAtvUSVmhul6So4C86tyh2q_J0-g-sdQqdkxqbxqamMZ7h13gdz6oM_tvv_dl6T8jsyeF4CSwmNyMbA1OqNCt7K6ogrROtdqnXNOZhIVdU6L4QvSOuWbVFUuOv5A-WC7ahYITH9Bvguwh9ga6jtGklpO1kYFHuqQLC1HiywxaBBBGaP_lCFjc1lG-7nvD90jJq3hypErzTJnZHP3EeTSGCN0c34ND0Pf7LTqlnJYTItAbP5zKKAsEJ_W63mPg
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< content-length: 385
< date: Tue, 15 Dec 2020 16:04:14 GMT
<
* Connection #0 to host localhost left intact
ReqData(Claims { sub: 3b08057d-782f-4b19-bd16-69ff219e783f, exp: 2024-02-15T01:54:32Z, realm_access: Some(Access { roles: ["test"] }), resource_access: Some({"client1": Access { roles: ["test1"] }, "client2": Access { roles: ["test2"] }}), iss: "actix-web-middleware-keycloak-auth", aud: Some("account"), iat: 2020-12-14T16:06:52Z, jti: 11111111-1111-1111-1111-111111111111, azp: "" })
```

```
$ cargo run --example simple
    Finished dev [unoptimized + debuginfo] target(s) in 0.91s
     Running `target\debug\examples\simple.exe`
[2020-12-15T16:03:33Z INFO  actix_server::builder] Starting 1 workers
[2020-12-15T16:03:33Z INFO  actix_server::builder] Starting "actix-web-service-127.0.0.1:8080" service on 127.0.0.1:8080
[2020-12-15T16:03:33Z TRACE actix_web_middleware_keycloak_auth] Initialize KeycloakAuthMiddleware
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth] Bearer token was extracted from request headers
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth] JWT header was decoded
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth] JWT is using RS256 algorithm
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth] JWT was decoded
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth::roles] JWT contains roles: test, client1.test1, client2.test2
[2020-12-15T16:04:14Z DEBUG actix_web_middleware_keycloak_auth] JWT is valid; putting claims in ReqData
[2020-12-15T16:04:14Z INFO  actix_web::middleware::logger] 127.0.0.1:54864 "GET /private HTTP/1.1" 200 385 "-" "curl/7.73.0" 0.003477
```

Did I forget something in my curl call?
Does it fail when you do the same?

**Comment by GaetanFauconnier85** [comment]
Sorry for the late answer. 
Yes when i try the same it fail.
`curl -v http://localhost:8080/private/users -H 'Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJ5akVsUy13SUZlQzNJcE9BZ2t6T1FmUXJGaERzTXI2VWgzeGZNYXpzemk0In0.eyJleHAiOjE2MTEzMDU1NTgsImlhdCI6MTYxMTMwNDQ3OCwianRpIjoiNzgwZjhhOTMtYWQzMi00ODBlLTlkZGQtNzlkZjJiMTMwYTY3IiwiaXNzIjoiaHR0cHM6Ly9pYW0uY2xvdWQtaWFtLmNvbS9hdXRoL3JlYWxtcy9zcG9ydGJvb2siLCJhdWQiOlsiY2xpZW50LWFwaS1saWtlIiwiYWNjb3VudCJdLCJzdWIiOiI3YzYxNWE0NS1mMWEwLTQ3YWMtYjZhMC1hODQ1NDJmNjNkZmMiLCJ0eXAiOiJCZWFyZXIiLCJhenAiOiJ0ZXN0IiwiYWNyIjoiMSIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJvZmZsaW5lX2FjY2VzcyIsImFkbWluIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJ0ZXN0Ijp7InJvbGVzIjpbInVtYV9wcm90ZWN0aW9uIiwibWVtYmVyIiwiYWRtaW4iXX0sImNsaWVudC1hcGktbGlrZSI6eyJyb2xlcyI6WyJhZG1pbiJdfSwiYWNjb3VudCI6eyJyb2xlcyI6WyJtYW5hZ2UtYWNjb3VudCIsIm1hbmFnZS1hY2NvdW50LWxpbmtzIiwidmlldy1wcm9maWxlIl19fSwic2NvcGUiOiJlbWFpbCBwcm9maWxlIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJjbGllbnRJZCI6InRlc3QiLCJjbGllbnRIb3N0IjoiNzcuMTUyLjIxMC4xMTMiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJzZXJ2aWNlLWFjY291bnQtdGVzdCIsImNsaWVudEFkZHJlc3MiOiI3Ny4xNTIuMjEwLjExMyJ9.NPjI_IMJGMR18hTCeFWDoNmloc99VQc_rsaTmWmgEvaRlsUNpdmm79gKhRONiGz-_Y00BsHWAetOnOapjy8NQkg8kjGiPr8PixNtUSSv4oIxJSsi0yEUKLCZYjx86uALeOdm2MqkZTgUnsujAqnpL5rwr1ZQ12E_-qP4Lop4SN12TtmTKM2zScGZicZ9LOWcaSkraiKcFJven5GBFrEZ_kmDdTnMd0HOGGelQZJtTCpM1md27OTrwUfkOSKjgKnBzIzDSJ2CIzSANLViqKoYVdEFsDYScoF8d0k5C-vWcM0OmXCBOif6EscbIYjHuHGLBOvUR4BDzO4233hEJQlSRg'
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 8080 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8080 (#0)
> GET /private/users HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.64.1
> Accept: */*
> Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJ5akVsUy13SUZlQzNJcE9BZ2t6T1FmUXJGaERzTXI2VWgzeGZNYXpzemk0In0.eyJleHAiOjE2MTEzMDU1NTgsImlhdCI6MTYxMTMwNDQ3OCwianRpIjoiNzgwZjhhOTMtYWQzMi00ODBlLTlkZGQtNzlkZjJiMTMwYTY3IiwiaXNzIjoiaHR0cHM6Ly9pYW0uY2xvdWQtaWFtLmNvbS9hdXRoL3JlYWxtcy9zcG9ydGJvb2siLCJhdWQiOlsiY2xpZW50LWFwaS1saWtlIiwiYWNjb3VudCJdLCJzdWIiOiI3YzYxNWE0NS1mMWEwLTQ3YWMtYjZhMC1hODQ1NDJmNjNkZmMiLCJ0eXAiOiJCZWFyZXIiLCJhenAiOiJ0ZXN0IiwiYWNyIjoiMSIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJvZmZsaW5lX2FjY2VzcyIsImFkbWluIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJ0ZXN0Ijp7InJvbGVzIjpbInVtYV9wcm90ZWN0aW9uIiwibWVtYmVyIiwiYWRtaW4iXX0sImNsaWVudC1hcGktbGlrZSI6eyJyb2xlcyI6WyJhZG1pbiJdfSwiYWNjb3VudCI6eyJyb2xlcyI6WyJtYW5hZ2UtYWNjb3VudCIsIm1hbmFnZS1hY2NvdW50LWxpbmtzIiwidmlldy1wcm9maWxlIl19fSwic2NvcGUiOiJlbWFpbCBwcm9maWxlIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJjbGllbnRJZCI6InRlc3QiLCJjbGllbnRIb3N0IjoiNzcuMTUyLjIxMC4xMTMiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJzZXJ2aWNlLWFjY291bnQtdGVzdCIsImNsaWVudEFkZHJlc3MiOiI3Ny4xNTIuMjEwLjExMyJ9.NPjI_IMJGMR18hTCeFWDoNmloc99VQc_rsaTmWmgEvaRlsUNpdmm79gKhRONiGz-_Y00BsHWAetOnOapjy8NQkg8kjGiPr8PixNtUSSv4oIxJSsi0yEUKLCZYjx86uALeOdm2MqkZTgUnsujAqnpL5rwr1ZQ12E_-qP4Lop4SN12TtmTKM2zScGZicZ9LOWcaSkraiKcFJven5GBFrEZ_kmDdTnMd0HOGGelQZJtTCpM1md27OTrwUfkOSKjgKnBzIzDSJ2CIzSANLViqKoYVdEFsDYScoF8d0k5C-vWcM0OmXCBOif6EscbIYjHuHGLBOvUR4BDzO4233hEJQlSRg
>
< HTTP/1.1 401 Unauthorized
< content-length: 107
< date: Fri, 22 Jan 2021 08:39:27 GMT
<
* Connection #0 to host localhost left intact
Error while decoding JWT token (JSON error: invalid type: sequence, expected a string at line 1 column 142)* Closing connection 0`

**Comment by dsferruzza** [comment]
Hi!

I believe I found the issue!
[The spec](https://tools.ietf.org/html/rfc7519#section-4.1.3) allows the `aud` claim to be a string _or_ a sequence of strings.
In my implement only a single string was accepted.

I fixed it in 010a23635eccda918a4b37d53278432bfb321f04; now `aud` is a `Vec<String>`.
I published a new version (`0.3.0`), can you give it a try and let me know?

_Also, next time please wrap your code/curl output in Markdown code blocks (```), it is pretty unreadable without this 😬_

**Comment by GaetanFauconnier85** [comment]
Hello,
This is working great now !!

```[2021-01-23T16:53:39Z DEBUG actix_web_middleware_keycloak_auth] JWT header was decoded
[2021-01-23T16:53:39Z DEBUG actix_web_middleware_keycloak_auth] JWT is using RS256 algorithm
[2021-01-23T16:53:39Z DEBUG actix_web_middleware_keycloak_auth] JWT was decoded
[2021-01-23T16:53:39Z DEBUG actix_web_middleware_keycloak_auth::roles] JWT contains roles: offline_access, admin, uma_authorization, realm-management.view-identity-providers, realm-management.view-realm, realm-management.manage-identity-providers, realm-management.impersonation, realm-management.realm-admin, realm-management.create-client, realm-management.manage-users, realm-management.query-realms, realm-management.view-authorization, realm-management.query-clients, realm-management.query-users, realm-management.manage-events, realm-management.manage-realm, realm-management.view-events, realm-management.view-users, realm-management.view-clients, realm-management.manage-authorization, realm-management.manage-clients, realm-management.query-groups, test.member, test.admin, account.manage-account, account.manage-account-links, account.view-profile, client-api-like.admin
[2021-01-23T16:53:39Z DEBUG actix_web_middleware_keycloak_auth] JWT is valid; putting claims in ReqData
```
Thanks for the fix =)
